Doc/library/sqlite3.rst

```diff
@@ -1353,8 +1354,6 @@ Connection objects
         implying that :mod:`!sqlite3` ensures a transaction is always open.
         Use :meth:`commit` and :meth:`rollback` to close transactions.
 
-        This is the recommended value of :attr:`!autocommit`.
```

Comment:
IMO, this line is causing too much confusion. For example, a lot of people seem to think `autocommit=True` is a bad thing, simply because `False` is the recommendation. I don't think it makes sense to make any recommendations. The programmer should be able to safely chose whatever value that makes the most sense for them based on how `autocommit` is documented.